docs/pages/try-out-teleport/local-kubernetes.mdx

```diff
@@ -125,8 +125,9 @@ HTTP API is working:
 
 ```code
 $ EXTERNAL_IP=$(kubectl get service teleport-cluster -o jsonpath='{ .status.loadBalancer.ingress[0].ip }')
-$ curl --insecure https://${EXTERNAL_IP?}:443/webapi/ping
-{"auth":{"type":"local","second_factor":"otp","preferred_local_mfa":"otp","has_motd":false},"proxy":{"kube":{"enabled":true,"listen_addr":"0.0.0.0:3026"},"ssh":{"listen_addr":"[::]:3023","tunnel_listen_addr":"0.0.0.0:3024","public_addr":"teleport-cluster:443"},"db":{"mysql_listen_addr":"0.0.0.0:3036"},"tls_routing_enabled":false},"server_version":"8.2.0","min_client_version":"7.0.0"}%
+$ TELEPORT_PING_URL="https://${EXTERNAL_IP?}:443/webapi/ping"
+$ curl --insecure $TELEPORT_PING_URL
+# {"auth":{"type":"local","second_factor":"otp","preferred_local_mfa":"otp","has_motd":false},"proxy":{"kube":{"enabled":true,"listen_addr":"0.0.0.0:3026"},"ssh":{"listen_addr":"[::]:3023","tunnel_listen_addr":"0.0.0.0:3024","public_addr":"teleport-cluster:443"},"db":{"mysql_listen_addr":"0.0.0.0:3036"},"tls_routing_enabled":false},"server_version":"8.2.0","min_client_version":"7.0.0"}%
```

Comment:
This long line is hard to read - maybe pipe it through jq?

Reply:
I generally agree, but we have to be careful about our assumptions of the reader's config; they may not have `jq` installed.

Reply:
We could omit the `| jq` from the example and display the formatted JSON in the docs page anyway. Not _exactly_ what they'd see if they ran it, but probably an improvement over this long line of JSON.

Reply:
I adjusted it to include the pipe but also explain it in the copy.